integration_tests/pages/page.ts

```diff
@@ -26,4 +26,12 @@ export default abstract class Page {
   signOut = (): PageElement => cy.get('[data-qa=signOut]')
 
   manageDetails = (): PageElement => cy.get('[data-qa=manageDetails]')
+
+  getByClass = (clazz: string): PageElement => cy.get(`.${clazz}]`)
```

Comment:
```suggestion
  getByClass = (class: string): PageElement => cy.get(`.${class}]`)
```
Though the original version still works 😀

Reply:
thats the cool why of saying it ;-)